workflow-core/src/commonMain/kotlin/com/squareup/workflow1/Worker.kt

```diff
@@ -273,7 +262,7 @@ public interface Worker<out OutputT> {
  */
 @OptIn(ExperimentalStdlibApi::class)
 public inline fun <reified OutputT> Flow<OutputT>.asWorker(): Worker<OutputT> =
-  TypedWorker(typeOf<OutputT>(), this)
+  TypedWorker({ OutputT::class.qualifiedName!! }, this)
```

Comment:
@rjrjr I'm getting rid of those gnarly "kotlin reflection not available"! At least for this one spot I found.